### DISCUSSION TITLE
[4.x]: Selectable Assets Condition - should be stackable?

### DISCUSSION THREAD

**Discussion by MattWilcox** [craftcms/cms#10991] [Ideas] [answered]
### Description

Selectable Assets Condition's are not stackable with the same parameter (once a parameter is used on one row, you can not select it again in another row).

This means it is not possible to set a condition that an Image asset should only be acceptable if its width (or height) are _between_ acceptable bounds. You may only enforce one bound.

Having had clients upload thumbnails as banner images and print-res images for thumbnails... I'd like an acceptable range.
TBH I'd also like a limit on the upload part of this system, so we never get 8000px wide images ingested in the first place.

<img width="995" alt="Screenshot 2022-04-22 at 09 52 09" src="https://user-images.githubusercontent.com/814039/164677038-3c05a6fa-0b85-4c22-a31f-962ac9f2a1e7.png">

**Comment by brandonkelly** [comment, marked as answer]
They’re single-use because they end up calling element query params (`width` in this case), which can only have a single value.

We can look into adding an “is between” option though, which would display two inputs, similar to date rules.

**Reply by brandonkelly** [reply]
Numeric condition rules have an “is between…” operator in Craft 4.3 (via #11950).

**Comment by torgeirbeyer** [comment]
Hi. Is there any plan on making it possible to use and/or in multiple Selectable Conditions? 

Example
Author: {{currentUser.id}} or Author Group is one of `Admin` ?

**Reply by brandonkelly** [reply]
“OR” conditions are planned for Craft 5. Not possible before then due to some technical limitations of element queries. You can follow along with the progress over at #11287.
